### DISCUSSION TITLE
是否可以使用HttpRequest发送multipart/form-data类型的POST请求？

### DISCUSSION THREAD

**Discussion by AS-N70** [looly/hutool#3043] [Q&A]
如题，想使用HttpRequest来发送一个Multipart/form-data，但是在给请求输入请求体的时候，body()方法总是会把我的请求体开头添加一些空格。请问该如何解决？即便我在body方法中指定了content-type为multipart/form-data;boundary=${boundary}
![image](https://user-images.githubusercontent.com/109048611/229663926-92b81170-a3c8-4040-a7bf-99ea2717014c.png)

**Comment by looly** [comment]
这个只是打印格式化，真正发送的时候并不会添加。

如果你是文件上传，通过form方法传入的值File对象时，Hutool自动将请求转换为multipart/form-data
